ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ConnectExecutor.java

```diff
@@ -47,6 +52,12 @@ public static StatementExecutorResponse execute(
     final CreateConnector createConnector = statement.getStatement();
     final ConnectClient client = serviceContext.getConnectClient();
 
+    final List<String> errors = validate(createConnector, client);
+    if (!errors.isEmpty()) {
+      return StatementExecutorResponse.handled(Optional.of(new ErrorEntity(
+          statement.getStatementText(), String.join("\n", errors))));
```

Comment:
Can we add a sentence to this error message to explain that the connector configuration failed validation? Otherwise the specific errors themselves are a bit hard to understand, without this addition context.

(Thanks for the additional test coverage for error messages, BTW. Makes it much easier to reason about these types of things :) )

Reply:
Done in 1bdc6f3.